src/store/reducers/index.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import UIReducer from './UIReducer';
 import articles from './article';
+import ResetPasswordReducer from './ResetPasswordReducer';
 
 export default combineReducers({
   UIReducer,
   articles,
+  reset_data: ResetPasswordReducer,
```

Comment:
What would the `reset_data` refer to in this Place?, I think a more descriptive name would help all of us in future

Reply:
I named it reset_data such that when I dispatch the payload from the actions I can be able to store them in the reset_data and use it later